### PR TITLE
Backport/RFE-748

### PR DESCRIPTION
**Pull request by shpran**
Backport: https://github.com/oat-sa/extension-lti-test-review/pull/137

### CODE REVIEW

**Review by hectoras — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "oat-sa/extension-tao-lti": ">=15.4.1",
         "oat-sa/extension-tao-delivery-rdf": ">=14.0.0",
         "oat-sa/extension-tao-proctoring": ">=20.0.0",
-        "oat-sa/extension-tao-testqti": ">=41.0.0",
+        "oat-sa/extension-tao-testqti": "dev-backport/rfe-748/45.0.6.2 as 45.0.6.2",
```
Comment (shpran) [inline composer.json:29]: - [ ] Change before merge

Comment (hectoras) [reply composer.json]: fcb73b0416c609661ebab6b04c46f4d42bb1da40

**Comment by hectoras** [comment]
Rebased to use [v3.8.2.1](https://github.com/oat-sa/extension-lti-test-review/releases/tag/v3.8.2.1) as the base version for these changes.

**Comment by hectoras** [comment]
Released as [v3.8.2.2](https://github.com/oat-sa/extension-lti-test-review/releases/tag/v3.8.2.2)

**Pull request merged**